pom.xml

```diff
@@ -41,6 +41,20 @@
       <artifactId>owasp-java-html-sanitizer</artifactId>
       <version>20180219.1</version>
     </dependency>
+    
+    <!-- https://mvnrepository.com/artifact/com.google.auto.value/auto-value -->
+    <dependency>
+      <groupId>com.google.auto.value</groupId>
+      <artifactId>auto-value</artifactId>
+      <version>1.7.3</version>
+      <scope>provided</scope>
```

Comment:
I don't know whether or not this is needed.

Documentation on this setting: https://maven.apache.org/guides/introduction/introduction-to-dependency-mechanism.html#dependency-scope

I wouldn't be surprised if `provided` is required due to the funny code generation that autovalue does. If apluscs@ got this from an example repo, I'd suggest keeping it. Perhaps just test compilation and usage without it?

Reply:
Compiled okay, l can test usage in another PR when making something like DatastoreAvailabilityDAO.